lumicks/pylake/fitting/fit.py

```diff
@@ -243,8 +256,8 @@ def fit(self, show_fit=False, **kwargs):
         if np.any(out_of_bounds):
             raise ValueError(
                 f"Initial parameters {self.params.keys()[fitted][out_of_bounds]} are outside the "
-                f"parameter bounds. Please set value, lower_bound and upper_bound for these parameters"
-                f"to consistent values."
+                "parameter bounds. Please set the model parameters, lower_bound and upper_bound "
```

Comment:
```suggestion
                "parameter bounds. Please set the model parameters lower_bound and upper_bound "
```
Is this comma needed?

Reply:
I think so? It's the `model parameters` (condition 1), `lower_bound` (condition 2) and `upper_bound` (condition 3).

Reply:
That is where I phased out for a moment... 🤦‍♂️